lib/server.js

```diff
@@ -34,15 +34,21 @@ module.exports = class HFServer {
       wsURL,
     })
 
+    const defaultOrders =  [AccumulateDistribute, TWAP, Iceberg, PingPong]
+    this.algoHost = new AOHost({
+      defaultOrders, apiKey, apiSecret, agent, wsURL, restURL
+    })
+
     this.as = new AOServer({
-      aos: [AccumulateDistribute, TWAP, Iceberg, PingPong],
+      aos: null,
```

Comment:
As mentioned in the other PR, pass `host: this.algoHost` instead of `aos: null` :slightly_smiling_face:

Reply:
see other pr comment